README.md

```diff
@@ -13,13 +13,13 @@
 1. [License](#license)
 
 ## Background
-ipfs-blockchain-watcher is a collection of interfaces that are used to extract, process, store, and index
-all blockchain data in Postgres-IPFS. The raw data indexed by ipfs-blockchain-watcher serves as the basis for more specific watchers and applications.
+ipld-eth-server is a collection of interfaces that are used to extract, process, store, and index
+all blockchain data in Postgres-IPFS. The raw data indexed by ipld-eth-server serves as the basis for more specific watchers and applications.
 
 Currently the service supports complete processing of all Bitcoin and Ethereum data.
 
 ## Architecture
-More details on the design of ipfs-blockchain-watcher can be found in [here](./documentation/architecture.md)
+More details on the design of ipld-eth-server can be found in [here](./documentation/architecture.md)
```

Comment:
This file does not exist

Reply:
Hey, yeah this is still WIP. Primarily the documentation that still needs updated.

Reply:
Ty for drawing attention to this though!